hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListenerCryptoTest.java

```diff
@@ -614,6 +615,7 @@ private void assertCryptoEntity(CryptoCreateTransactionBody expected, Timestamp
         Entity actualProxyAccountId = getEntity(actualAccount.getProxyAccountId());
         assertAll(
                 () -> assertEquals(expected.getAutoRenewPeriod().getSeconds(), actualAccount.getAutoRenewPeriod()),
+                () -> assertNotNull(actualAccount.getCreatedTimestamp()),
```

Comment:
We should assert the exact value. Since this method is only used by creates it should be okay to do:
```suggestion
                () -> assertEquals(consensusTimestamp , actualAccount.getCreatedTimestamp()),
```

Reply:
made the change, also removed the invalid test case.